mern/client/src/screens/UploadTranscript/uploadTranscript.js

```diff
@@ -80,13 +106,34 @@ function UploadTranscript() {
           buttonType="outline"
           onClick={() => {
             PageChange("/");
-            setNavState(false)
+            setNavState(false);
           }}
           disabled={false}
           text={"Go Back"}
         />
-        {openSuccessModal && (
-          <SuccessModal closeModal={setOpenSuccessModal} fileName = {fileName} />
+        {setShowModal && (
+          <Modal
+            show={showModal}
+            title="Name your flow to begin"
+            body="Enter your flow name"
+            value={flowName.name}
+            onChange={handleFlowNameChange}
+            onClose={() => {
+              setShowModal(false);
+              deleteFile(transcriptID);
+            }}
+            onSubmit={() => {
```

Comment:
the uploadTranscript file has gotten a bit long (I know the generic components take up a good line count). is there anything we can extract in seperate files?

Reply:
You're right. Need to explore splitting them into different components that are called in a new file